### ISSUE TITLE
Pixel 7/7 Pro freezes when repeating task switching while playing widevine content

### ISSUE THREAD

**Issue by kjmkznr** [google/ExoPlayer#11059]
### ExoPlayer Version

2.18.2

### Devices that reproduce the issue

* Pixel 7 running on Android 13 (TQ1A.230205.002)
* Pixel 7 Pro running on Android 13

### Devices that do not reproduce the issue

* Pixel 6 running on Android 13 (TQ1A.230205.002)

### Reproducible in the demo app?

Yes

### Reproduction steps

1. Play `Widevine DASH (MP4, H264 > HD (cenc)` video
2. Until displayed recents screen swipe up the bottom of the screen
3. Tap ExoPlayer demo app
4. Repeat step 2 to 3

### Expected result

Continuous playback without freeze

### Actual result

Freeze UI and reboot device

### Media

`Widevine DASH (MP4, H264) > HD (cenc)`

### Bug Report

- [X] You will email the zip file produced by `adb bugreport` to dev.exoplayer@gmail.com after filing this issue.

**Comment by rrfrias** [comment]
Sorry about the delay. I have opened up internal tracker [283039759](http://b/283039759) nd requested access to the bugreport. Do grant me access when you get a chance. Thanks!

**Comment by rrfrias** [comment]
Thanks! I got the bugreport. I see a number of ANRs (Application Not Responsive) but i do not see them captured in the log in the bugreport. Can you capture a logcat and share that us. Email it to [dev.exoplayer@gmail.com](mailto:dev.exoplayer@gmail.com) with subject "Issue 11059". We will try and reproduce as well.

**Comment by rrfrias** [comment]
We have been able to reproduce. The logcat is no longer needed.

**Comment by rrfrias** [comment]
We are waiting on an internal team in android to investigate.

**Comment by rrfrias** [comment]
Still waiting on the internal team to take a look

**Comment by rrfrias** [comment]
Still waiting on the internal team to investigate.

**Comment by rrfrias** [comment]
This issue was fixed and included in Pixel builds on/after December 2023 (android 14) starting with build string UQ1A.
